lib/team_data.rb

```diff
@@ -11,12 +12,12 @@ def add_teams
     teams = CSV.open './data/teams.csv', headers: true, header_converters: :symbol
     teams.each do |row|
       team_id = row[:team_id]
-      franchiseid = row[:franchiseId]
-      teamname = row[:teamName]
+      francise_id = row[:franchise_id]
```

Comment:
Way to chop the sneaky caps down to size

Reply:
good catch!